templates/api.hbs

```diff
@@ -1,17 +1,39 @@
 {{>header}}
 
-import { Express } from 'express'
+import { Express, Request, Response } from 'express'
 import passport from 'passport'
+import multer from 'multer'
 import * as t from './types'
 import * as v from '../../validation'
+{{#if (hasOperationSupportingMultipart operations)}}
+import * as f from '../../impl/helpers/{{lowerCase name}}MultipartHelper'
+{{/if}}
 import { Api } from '../../models'
 
 export default function(app: Express, impl: t.{{className name}}Api) {
 	{{#each operations}}
 	app.{{lowerCase httpMethod}}(
 		{{{stringLiteral (pathTemplate fullPath)}}},
 		{{>frag/apiSecurityRequirements}}
-		function (req, res) {
+		{{#if (hasOperationSupportingMultipart .)}}
```

Comment:
Curious that we don't need to check for multipart, we only need to check for `FILE` as they're always multipart. That's curious because they're sort of separate concepts... it's not clear that `FILE` is only multi-part, but we set the type to `Express.Multer.File` in our `index.ts`... HMM